src/EnergyPlus/WaterUse.cc

```diff
@@ -225,19 +216,18 @@ namespace WaterUse {
 
         if (state.dataGlobal->BeginEnvrnFlag && this->MyEnvrnFlag) {
             if (state.dataWaterUse->numWaterEquipment > 0) {
-                for (int i = state.dataWaterUse->WaterEquipment.l(), e = state.dataWaterUse->WaterEquipment.u(); i <= e; ++i) {
-                    state.dataWaterUse->WaterEquipment(i).reset();
-
-                    if (state.dataWaterUse->WaterEquipment(i).setupMyOutputVars) {
-                        state.dataWaterUse->WaterEquipment(i).setupOutputVars(state);
-                        state.dataWaterUse->WaterEquipment(i).setupMyOutputVars = false;
+                for (auto &waterEquipment : state.dataWaterUse->WaterEquipment) {
+                    waterEquipment.reset();
+                    if (waterEquipment.setupMyOutputVars) {
+                        waterEquipment.setupOutputVars(state);
+                        waterEquipment.setupMyOutputVars = false;
                     }
                 }
             }
 
             if (state.dataWaterUse->numWaterConnections > 0) {
-                for (auto &e : state.dataWaterUse->WaterConnections)
-                    e.TotalMassFlowRate = 0.0;
+                for (auto &waterConnections : state.dataWaterUse->WaterConnections)
+                    waterConnections.TotalMassFlowRate = 0.0;
```

Comment:
Although braces aren't necessary here, I still like to have them.  In this case, it would be easy for someone to add another calculation to the next line and not realize they aren't adding it in that scope.  In that case, it may cause a compiler error due to variable scope, or compiler warning if the compiler figures out what happened, but still, braces are intentional and explicit.  Not necessary to change now unless there are other changes to be made.